easybuild/easyconfigs/l/libXext/libXext-1.3.3-intel-2015b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'libXext'
+version = '1.3.3'
+
```

Comment:
same here, we already have libXext-1.3.3-intel-2015a-Python-2.7.10.eb, so keep the versionsuffix?